doc/users/text_props.rst

```diff
@@ -59,3 +60,97 @@ upper right.
 
 .. plot:: pyplots/text_layout.py
    :include-source:
+
+
+==============
+ Default Font
+==============
+
+The base default font is controlled by a set of rcParams
```

Comment:
I guessed that is what it was, but this was right above a table, so 'column' also made literal sense.